client/core/trade.go

```diff
@@ -1085,7 +1085,7 @@ func (t *trackedTrade) revoke() {
 		t.dc.log.Errorf("unable to update order: %v", err)
 	}
 
-	t.notify(newOrderNote("revoke", "", db.Data, t.coreOrderInternal()))
+	t.notify(newOrderNote("revoke", "", severity, t.coreOrderInternal()))
```

Comment:
This was a `Data`-`Severity`notification, which is why there is no message and the subject is not capitalized. [`Data` notifications are not meant for display](https://github.com/decred/dcrdex/blob/dfbb3b334d740dd353d6d23237aad03fde9f9904/client/db/types.go#L32). If you set the severity to `WarningLevel`, `ErrorLevel`, or `Success`, the user will get a UI notification (the bell) with no details and just the work "revoke" for a subject.

The caller needs to send the notification themselves, with custom messaging and severity. Once **all** of the callers are sending notifications, we can actually remove this line. This was just here to force an update of the browsers cached order data, but any `OrderNote` will do.

Reply:
Callers are sending notifications now.